### ISSUE TITLE
NVDA does not read a new window open

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#5699]
When you open some windows NVDA does not inform the window open, this occurs many times to open documents in Word.
For the window to be read it is necessary to remove the window focus with alt + tabe and return the focus to alt + tabe again.
 This does not happen all times but occurs very frequently.
 In the log file attached there is a demonstration of the problem.
 Hall reproduce to the problem.
 1 open a document in Word.
 NVDA is expected to read the document.
 What happens is that NVDA does not read anything, being necessary to remove the window focus with alt + tabe and return with alt + tabe again for the document to be read.
[word nvda.log.txt](https://github.com/nvaccess/nvda/files/101243/word.nvda.log.txt)

**Comment by jcsteh** [comment]
When you say "open a document in Word", do you mean by pressing 
control+o in Word or opening the document from File Explorer?

**Comment by fernando-jose-silva** [comment]
I open the word from the windows explorer.

Open the start menu item this computer, choose the drive and open, then choose the folder where the .docx file, and finally Press enter on the .docx file.

In doing so NVDA informs word, but does not read the file with the mouse, it is necessary to read the file to remove the focus from the word with alt + Tavern and return the focus to alt + Tavern again.

This problem does not occur every time.

This problem also occurs with some other windows, and it's not very frequently.

thank you

Open the start menu item this computer, choose the drive and open, then choose the folder where the .docx file, and finally Teclo enter on the .docx file.

In doing so NVDA informs word, but does not read the file with the mouse, it is necessary to read the file to remove the focus from the word with alt + Tavern and return the focus to alt + Tavern again.

This problem does not occur every time.

This problem also occurs with some other windows, and it's not very frequently.

thank you

 

De: James Teh [mailto:notifications@github.com] 
Enviada em: segunda-feira, 25 de janeiro de 2016 01:08
Para: nvaccess/nvda <nvda@noreply.github.com>
Cc: fernando-jose-silva <fernandoj.silva21@gmail.com>
Assunto: Re: [nvda] NVDA does not read a new window open (#5699)

 

When you say "open a document in Word", do you mean by pressing 
control+o in Word or opening the document from File Explorer?

**Comment by Adriani90** [comment]
Closing in favor of #8389
